CHANGELOG.md

```diff
@@ -1,217 +1,107 @@
-Release Notes
-====
+# Release Notes
 
-##### Newer releases
-
-See [GitHub Releases](https://github.com/winsw/winsw/releases)
+See [Releases](https://github.com/winsw/winsw/releases) and [Milestones](https://github.com/winsw/winsw/milestones).
 
 ##### 2.3.0
 
-Release date: Aug 19, 2019
-
-* [PR #315](https://github.com/kohsuke/winsw/pull/315) - 
-Prevent application crash when `roll-by-size-time` is enabled
-* [PR #321](https://github.com/kohsuke/winsw/pull/321) - 
-Add a new `hidewindow` option to suppress windows popup on legacy platforms 
-* [PR #330](https://github.com/kohsuke/winsw/pull/330) -
-Fix duplicate startup log entries
-* [PR #299](https://github.com/kohsuke/winsw/pull/299),
-  [PR #303](https://github.com/kohsuke/winsw/pull/303),
-  [PR #311](https://github.com/kohsuke/winsw/pull/311),
-  [PR #316](https://github.com/kohsuke/winsw/pull/316),
-  [PR #318](https://github.com/kohsuke/winsw/pull/318) -
-  Documentation cleanup
+* Prevent application crash when `roll-by-size-time` is enabled (#315)
+* Add a new `hidewindow` option to suppress windows popup on legacy platforms (#321)
+* Fix duplicate startup log entries (#330)
+* Documentation cleanup (#299, #303, #311, #316, #318)
 
-Internal changes:
+## Internal changes
```

Comment:
Wrong header levels here and below. Version header is h5 while this one is h2

Reply:
So I'm going to delete these since they're now present in Releases.

Reply:
Fine with me